.github/workflows/upload-and-deploy.yaml

```diff
@@ -202,6 +206,13 @@ jobs:
           aws_secret_access_key: ${{ secrets.CI_USER_SECRET_ACCESS_KEY }}
           role_to_assume: ${{ secrets.CI_ROLE_TO_ASSUME }}
 
+      - name: Copy files to templates bucket
```

Comment:
I believe you mentioned that in BD we share the `v0.1` namespace. Is this section just setting us up for the future when we use the namespaced scripts?

Reply:
No, this just propagates the existing design of having the Glue job scripts put in the v0.1 namespace and everything else being put in the "actual" namespace.